### DISCUSSION TITLE
Description vs. Comment

### DISCUSSION THREAD

**Discussion by googol7** [community/community#61826] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hi GitHub,

In the documentation you write this:

https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

> You can link a pull request to an issue by using a supported keyword in the pull request's description or in a commit message.

The interface doesn’t know a "description" because it’s called "comment" and the button is called "Update comment". The placeholder text is "Leave a comment".

<img width="924" alt="Github description" src="https://github.com/community/community/assets/282412/4baaa4cf-7b9f-4c6d-a0ef-30090204e866">

This is really confusing. I think you should call it "description" like in the documentation so it can be distinguished from comments below.

"fix [#123](https://enterprise.githubsupport.com/hc/requests/123)" only closes an issue when a pull request is merged when entered in the description but not when entered in a comment.

What do you think?

Philipp

**Comment by moonfury-ops** [comment]
Thank you

**Comment by faisalnazir7** [comment]
**Description:**

Repository Description: When you create a new repository on GitHub, you have the option to provide a short "Description" for the repository. This description appears on the repository's main page and provides a brief overview of what the repository is about or what it contains. It is useful for visitors and potential contributors to quickly understand the purpose of the repository.

Pull Request Description: When creating a pull request to propose changes to a repository, you can include a "Description" for the pull request. This description serves as a summary of the changes being proposed and can include additional context, references, or explanations related to the pull request. It helps reviewers and collaborators understand the purpose and scope of the changes.

 **Comment:**

Issue and Pull Request Comments: Once an issue or a pull request is created, users (including collaborators and contributors) can add comments to provide feedback, ask questions, or discuss specific aspects of the issue or pull request. Comments are useful for having threaded discussions and collaboration among team members. They allow multiple participants to share their thoughts and ideas related to the topic at hand.

**Comment by vazhnov** [comment]
@googol7 , am I right you are writing here that in a merge request, first comment is actually a description, not a comment?
If yes, I support you.
Also, for example, everyone in a team can edit the first comment (which is description), and this behavior looks weird for a comment.
